### ISSUE TITLE
Details/Expander/Expander group html markup

### ISSUE THREAD

**Issue by nnekaO** [nhsuk/nhsuk-frontend#480]
The markup for these are different from the html rendered in the respective examples. The markup is missing attributes like aria-expanded="false" and aria-hidden="true"
